OpenDreamRuntime/DreamManager.Connections.cs

```diff
@@ -17,6 +19,18 @@ sealed partial class DreamManager
 
         public DreamConnection GetConnectionBySession(IPlayerSession session) => _connections[session];
         public IEnumerable<DreamConnection> Connections => _connections.Values;
+        /// <summary> Returns the port that the server is open on. </summary>
+        public int Port => _netManager.Port;
+
+        /// <summary> Tries to return the address of the server, as it appears over the internet. May return null.</summary>
+        public IPAddress? InternetAddress
+        {
+            get
+            {
+                NetManager net = (NetManager)_netManager; // idfk man
```

Comment:
Maybe worth noting that `IServerNetManager` has another implementation, `IntegrationNetManager`. I think that's used in integration tests, and this would not work there.